test_fixtures/transformer_qa/experiment.jsonnet

```diff
@@ -0,0 +1,48 @@
+local transformer_model = "albert-base-v2";
```

Comment:
Is this the smallest model you can find?

Reply:
Yes. @ZhaofengWu found it actually.

It's still too big to check in a trained model, but it makes the test run faster if this is small.